### PR TITLE
Oai gbr xapp tutorial

### PR DESCRIPTION
**Pull request by EugenioMoro**
this pr is to include a tutorial on SLA management using an xapp and oai on colosseum

### CODE REVIEW

File: _data/publications.yml [hunk 1]
```diff
@@ -160,6 +160,15 @@ publications:
...
 
+  - name: moro2023open
+    author: E. Moro, M. Polese, A. Capone, T. Melodia
+    title: "An Open RAN Framework for the Dynamic Control of 5G Service Level Agreements"
+    booktitle: Proceedings of IEEE NFV-SDN
+    address: Dresden, Germany
+    month: November
+    year: 2023
+    pdflink: "https://arxiv.org/pdf/2309.07508.pdf"
+    bibtexlink: "https://scholar.googleusercontent.com/scholar.bib?q=info:mLiNmrrMUN8J:scholar.google.com/&output=citation&scisdr=ClGyTutsEKi15M7ENgs:AFWwaeYAAAAAZWXCLgvZiaNQZCfEMz0gC92Exq0&scisig=AFWwaeYAAAAAZWXCLjK7VaUZ56EZYzF2TTQnqmY&scisf=4&ct=citation&cd=-1&hl=en"
```
Comment (leonardobonati) [inline _data/publications.yml:171]: The bibtex link gives me an error.  I would suggest to use this one: https://ece.northeastern.edu/wineslab/wines_bibtex/moro2023nfv.txt

File: _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: "Managing GBR SLA with an O-RAN xApp and OAI"
+date: 2022-11-02
```
Comment (leonardobonati) [inline _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md:4]: IIRC the page puts the tutorials in chronological order. Do you want to update the date here with a current one, matching the one in the name of the file?

File: _data/publications.yml (hunk 1 above)
Comment (leonardobonati) [reply _data/publications.yml:171]: Also please let's try to maintain the publication in chronological order

**Review by leonardobonati — CHANGES_REQUESTED** [review]

File: _data/publications.yml (hunk 1 above)
Comment (EugenioMoro) [reply _data/publications.yml]: sorry, I corrected the issue

**Comment by mychele** [comment]
Let's keep the old page and have everything there, otherwise every link (from the outside) to the OAI tutorial will break @EugenioMoro

File: _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md (hunk 2 above)
Comment (leonardobonati) [reply _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md:4]: I would still suggest to have the filename and the date field reflecting the same date, you choose which one :)

**Comment by leonardobonati** [comment]
> Let's keep the old page and have everything there, otherwise every link (from the outside) to the OAI tutorial will break @EugenioMoro

We can also put a redirect in case.

File: _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md (hunk 2 above)
Comment (EugenioMoro) [reply _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md:4]: I'm matching the date here with the filename

**Comment by EugenioMoro** [comment]
I've fixed the date as requested by Leo, I believe it is ready now

File: _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+{% assign publications = site.data.publications %}
+
+{% assign el = publications.publications | where: "name", "moro2023open" %}
+{% assign element = el[0] %}
+{% capture pub_moro2023open %}
+{% include pub-template.html %}
+{% endcapture %}
+
+OpenAirInterface (OAI) is a 3GPP-compliant software implementation of the full 5G NR stack. In this tutorial, we show how OpenRAN Gym users can get started with deploying an OAI 5G NR SA network composed of a gNB and a few UEs on Colosseum. Additionally, this tutorial shows how to deploy a nRT-RIC and an xApp developed to manage GBR SLAs of the RAN UEs.
```
Comment (leonardobonati) [inline _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md:18]: Should we define the acronym `GBR`?

File: _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+python3 elastic_sla.py
+{% endhighlight %}
+
+Finally, we can generate downlink UE traffic as follows. In every UE container, add the route to the core network host:
+
+{% highlight bash %}
+ip route add 192.168.70.128/26 via 12.1.1.1
+{% endhighlight %} 
+
+and take note of the UE GTP endpoint IP:
```
Comment (leonardobonati) [inline _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md:118]: Should we define the acronym `GTP`?

**Review by leonardobonati — COMMENTED** [review]
Looks good to me. Should we define the acronyms `GBR` and `GTP`?

File: _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md (hunk 3 above)
Comment (EugenioMoro) [reply _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md:18]: Done, I've also defined SLA

File: _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md (hunk 4 above)
Comment (EugenioMoro) [reply _posts/tutorials/2023-11-28-oai-o-ran-rrm-xapp.md:118]: Done

**Comment by leonardobonati** [comment]
Thanks, ok to merge. @mychele?

**Review by leonardobonati — APPROVED** [review]

**Comment by mychele** [comment]
LGTM, at some point we should update with the new OAI images as these ones will not work well on the updated SRNs.

**Pull request merged**